### DISCUSSION TITLE
2022/08/nas-upgrade/

### DISCUSSION THREAD

**Discussion by giscus** [Li-Aaron/Li-Aaron.github.io#3] [Announcements]
# 2022/08/nas-upgrade/

本文介绍黑裙升级CPU主板，系统信息更换正确CPU名称，以及开启video station中truehd/dts/ac3音频解码的方法。

https://li-aaron.github.io/2022/08/nas-upgrade/

**Comment by Li-Aaron** [comment]
测试评论区
